app/views/pledges/index.html.erb

```diff
@@ -1,5 +1,10 @@
 <div class="container">
-  <p id="notice"><%= notice %></p>
+  <% if notice %>
+    <div class="alert alert-success alert-dismissable">
```

Comment:
It looks like everything is using `alert-success` right now. Have you considered using different types of alerts, like `alert-danger` or `alert-warning` (when a user tries to go to an unauthorized page, model validations failed, etc.?)